src/flisp/flisp.c

```diff
@@ -2485,6 +2485,10 @@ int fl_load_system_image(fl_context_t *fl_ctx, value_t sys_image_iostream)
     return 0;
 }
 
+#ifndef _OS_WINDOWS_
```

Comment:
Should this be guarding on `HAVE_VASPRINTF` instead?

Also, why did this come up as part of this change?

Reply:
I think you mean `HAVE_ASPRINTF`, but we never define that as far as I can tell. Instead `asprintf.c` is included if we're on windows.

This has come up as we only define asprintf on windows, so to ensure its not undefined it needs to be set.